newIDE/app/src/Utils/GDevelopServices/Asset.js

```diff
@@ -25,12 +25,8 @@ export type AssetCustomization =
       properties: Array<SerializedParameterMetadata>,
     |}
   | {|
-      required: boolean,
-      parameters: Array<SerializedParameterMetadata>,
-      extensions: Array<{|
-        extensionName: string,
-        extensionVersion: string,
-      |}>,
+      extensionName: string,
+      extensionVersion: string,
```

Comment:
I wonder if `AssetCustomization` is still a good name.
Should behavior properties be in the serialized object? In this case, it would just be a list of required extensions.
Behaviors on assets is a dead code. Should it be removed?

Reply:
I guess the idea with assets customisation was to offer OPTIONAL behaviors that would be interesting for the object... But this seems outdated/useless. 
So we could rename this (just be sure not to crash if the existing assets don't have the new property key).

Reply:
I guess this can be.. requiredExtensions?

Reply:
ok, I renamed it. I'll have to report it to the asset repository too.